### ISSUE TITLE
Wrap With Span option

### ISSUE THREAD

**Issue by Spellhammer** [soflyy/oxygen-bugs-and-features#634]
Posted on behalf of Tobias Alriksson.

My understanding is that this would function the exact same way as the current "Wrap With Div" option does, except it would generate a <span> element instead.

**Comment by wplit** [comment]
Couldn't you just add <span> as one of the tag options for divs?

**Comment by alriksson** [comment]
@wplit Yes correct but would like an option to wrap things in `<span>` to style things but reduce the `<div>`s

So it could be as you say both in the tree view "wrap in div or wrap in span" and as a drop-down tag option.
